lib/sapiV1CapiV2.js

```diff
@@ -486,10 +486,16 @@ function getCapi(uuid) {
 	    ;
 }
 
-const concertinaOverlapThreshold = 0.5;
 const concertinaNameJoin = ' + ';
 
-function concertinaSortedListAndUpdateUuidGroups( sortedList, uuidsGroupedByItem ){
+function concertinaSortedListAndUpdateUuidGroups( sortedList, uuidsGroupedByItem, params={} ){
+  const defaultParams = {
+    concertinaOverlapThreshold : 0.66,
```

Comment:
magic number?

Reply:
a default value that is override-able as a url param

Reply:
still experimenting. Could make it a env param, but not worth the hassle yet.